posthog/api/test/batch_exports/test_test.py

```diff
@@ -261,7 +261,14 @@ def test_can_run_snowflake_test_step_for_partial_config(client: HttpClient, snow
             batch_export_data,
         )
 
-        batch_export_data["destination"]["config"] = {"account": "Something", "authentication_type": "password"}
+        batch_export_data = {
+            "name": "my-production-snowflake-destination",
+            "destination": {
+                "type": "Snowflake",
+                "config": {"account": "Something", "authentication_type": "password"},
+            },
+            "interval": "hour",
+        }
```

Comment:
Not sure why mypy was crying about the above line. You can clearly see that 10-20 lines above `batch_export_data["destination"]["config"]` is set and thus can be set again.